storage/migrations/01_consensus.up.sql

```diff
@@ -280,8 +280,14 @@ CREATE TABLE oasis_3.processed_blocks
   PRIMARY KEY (height, analyzer)
 );
 
+-- Keeps track of chains for which we've already processed the genesis data.
+CREATE TABLE chain.processed_geneses (
+    chain_context TEXT NOT NULL PRIMARY KEY,  -- identifies the genesis data; derived from its hash
+    processed_time TIMESTAMP WITH TIME ZONE NOT NULL
```

Comment:
We removed the `analyzer` field because only consensus will ever have a genesis document that we'll process?

Reply:
Yes! Runtimes don't really have a genesis doc.
There is a genesis struct that runtimes can specify with every new version, but it's hardcoded into the binary (IIRC) and opaque, so I think the best we can do there is manually encode it in indexer, should a significant genesis ever come out. The ones so far have been no-ops.